### DISCUSSION TITLE
Adding custom blocks to Navigation menus

### DISCUSSION THREAD

**Discussion by paaljoachim** [WordPress/developer-blog-content#357] [Topic Ideas]
Hey

I posted in the Make WordPress Slack channel for feature website navigation:
https://app.slack.com/client/T024MFP4J/C01KDAZJMQ9

It would be great to have a code snippet to where one can add additional blocks to the Navigation menus.
@ndiego 

Thanks.

**Comment by ndiego** [comment]
Closing in favor of #355
